src/app/home/shared/forms/transaction-form/dtos.ts

```diff
@@ -0,0 +1,18 @@
+import {TimestampDto} from '../../../../shared/dtos';
+
+export interface NewTransactionRequest {
+  name: string;
+  amount: number;
+  date: Date;
+  budgetId: string;
+  uid: string;
+}
+
+export interface TransactionDto {
+  name: string;
+  amount: number;
+  date: TimestampDto;
+  budgetId: string;
+  uid: string;
+  id: string;
+}
```

Comment:
You can reduce this type, to only contain props that are different than NewTransactionRequest:

```ts
export interface TransactionDto extends Omit<NewTransactionRequest, 'date'>  {
  date: TimestampDto;
  id: string;
}
```

Reply:
Ok